code/src/index.css

```diff
@@ -1,6 +1,14 @@
+html {
+  box-sizing: border-box;
+}
```

Comment:
Does it work this way too? Usually I either write 
```
* {
  box-sizing: border-box;
}
```
or
```
*, *:before, *:after {
  box-sizing: border-box;
}
```

Reply:
It worked... I think 😅 but yes, the * is probably better to use. Not sure about the _before, and _after. Maybe you can tell me sometimes.